### PR TITLE
some modification in make_install.bat

### PR DESCRIPTION
**Pull request by freemine**
1. copy files failed if svc-deleting is still outstanding
2. unnecessary duplication of C:\TDengine in %PATH%

### CODE REVIEW

File: packaging/tools/make_install.bat [hunk 1]
```diff
@@ -105,3 +113,36 @@ rem // use setx to set a temporary throwaway value to trigger a WM_SETTINGCHANGE
...
+
+:stop_delete
+sc stop taosd
+sc delete taosd
+sc stop taosadapter
+sc delete taosadapter
+exit /B 0
+
+:check_svc
+sc query %1 >nul 2>nul && goto :check_svc %1
```
Comment (facetosea) [inline packaging/tools/make_install.bat:145]: Is there any situation here that will cause an infinite loop?

**Comment by freemine** [comment]
please comment @sangshuduo @facetosea

**Comment by facetosea** [comment]
@freemine I am worried that check_svc function may have an infinite loop, other changes I think no problem

**Comment by freemine** [comment]
@facetosea, thx.

case1: there's no registered `taosd` service, `sc query taosd` would fail and break out the loop
case2: `taosd` is running, this could not happend, because `sc stop taosd` has just been called previously
case3: `taosd` is stopping, `sc query taosd` would success with status and thus, continue the loop
case4: once `taosd` is stopped, deletion of `taosd` will complete, and switch to `case1`

it seems the only way to make the loop infinite is `taosd` does not yield execution when being requested to stop, which, optimistically, is believed not to happen. :)

**Review by facetosea — APPROVED** [review]
LGTM

**Pull request merged**